tests/fib/test_fib.py

```diff
@@ -177,6 +177,11 @@ def setup_hash(tbinfo, duthosts, rand_one_dut_hostname, config_facts):
     if duthost.facts['asic_type'] in ["barefoot"]:
         if 'ingress-port' in hash_keys:
             hash_keys.remove('ingress-port')
+    if duthost.facts['platform'] in ['armhf-nokia_ixs7215_52x-r0']:
```

Comment:
please add comment, why remove ?

Reply:
@Iguohan 
I added comment since for this platform these keys for hashing are not supported  we need to remove them for running ip hash testing